template/App.js

```diff
@@ -6,44 +6,85 @@
  * @flow
  */
 
-import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
+import React, {Fragment} from 'react';
+import {StyleSheet, ScrollView, View, Text, StatusBar} from 'react-native';
 
-const instructions = Platform.select({
-  ios: 'Press Cmd+R to reload,\n' + 'Cmd+D or shake for dev menu',
-  android:
-    'Double tap R on your keyboard to reload,\n' +
-    'Shake or press menu button for dev menu',
-});
+import Header from 'react-native/Libraries/NewAppScreen/components/Header';
```

Comment:
I'd like to pull these up into one destructed import like @orta mentioned, but I noticed that all the modules that RN exposes here are all default exports like we have here already. Is this worth doing as a part of this PR or is this something worth discussing elsewhere?

Reply:
I think I'd prefer importing this from "NewAppScreen" instead of reaching deep inside the folder.